common/types/mapper/thrift/configStore_test.go

```diff
@@ -0,0 +1,156 @@
+package thrift
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
```

Comment:
files need the license thing at the top and imports follow certain ordering. if you run `make fmt` it should take care of these.

Reply:
updated the file. Can you please check once.

Reply:
You'll also need to run
make fmt.
We use gci to ensure imports sort/grouping/order.

Reply:
yeah added the license info and ran make fmt command also.